fast-socket-installer/image/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM k8s.gcr.io/debian-base:v1.0.0
+FROM gcr.io/gke-release/debian-base:latest
```

Comment:
Use a specific digest or tag instead of latest, otherwise things could be broken unexpectedly.

Reply:
I see. I changed to the newest tag.

Reply:
I see. I just changed it to the newest tag.